demo/shared/media/twitter.inline.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I suggest optimizing this icon to show the best practice
https://jakearchibald.github.io/svgomg/